### PR TITLE
Add common API and UI methods in  separate modules

### PR DESCRIPTION
**Pull request by aaujayasena**
## Purpose

Introduce two modules for API and UI.  API module contain the common API functions available on identity server and third party provided APIs. UI module contain the common UI artifacts such as page objects and page constant available on identity server 
 
## Goals
Having two separate module will help to prevent unnecessary import to test project. 

## sample
Module import
import "@wso2/identity-cypress-test-base/ui"
import "@wso2/identity-cypress-test-base/api"

### CODE REVIEW

File: src/api/commands/scim2.0-user-managment.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (brionmario) [inline src/api/commands/scim2.0-user-managment.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

File: api.js [hunk 2]
```diff
@@ -17,4 +17,5 @@
  *
  */
 
-module.exports = require("./dist/utils");
+module.exports = require("./dist/api");
+
```
Comment (brionmario) [inline api.js:21]: Seems like there is an extra line break here. Please validate.

File: ui.js [hunk 3]
```diff
@@ -17,4 +17,5 @@
  *
  */
 
-module.exports = require("./dist/commands");
+module.exports = require("./dist/ui");
+
```
Comment (brionmario) [inline ui.js:21]: Seems like there is an extra line break here. Please validate.

**Review by brionmario — APPROVED** [review]

File: src/api/commands/scim2.0-user-managment.ts (hunk 1 above)
Comment (aaujayasena) [reply src/api/commands/scim2.0-user-managment.ts]: Fixed with https://github.com/wso2/identity-cypress-test-base/pull/13/commits/99af09763311ab7271e55b85b9dbd83d84f2c112

File: api.js (hunk 2 above)
Comment (aaujayasena) [reply api.js]: Fixed with https://github.com/wso2/identity-cypress-test-base/pull/13/commits/99af09763311ab7271e55b85b9dbd83d84f2c112

File: ui.js (hunk 3 above)
Comment (aaujayasena) [reply ui.js]: Fixed with https://github.com/wso2/identity-cypress-test-base/pull/13/commits/99af09763311ab7271e55b85b9dbd83d84f2c112

**Pull request merged**